_view/attachChildViews.md

```diff
@@ -3,4 +3,4 @@ header: attachChildViews
 example: View.attachChildViews()
 ---
 
-Removes all child views, then looks at all of the child views that are expected and attaches each view.
+Removes all child views, then looks at all of the expected child views and attaches each view.
```

Comment:
yeah - it checks the DOM (the filtered to just the $el) for an element with `[data-view]` which is attached by the templateAdapter view helper.